tests/integration/models/gpflux/conftest.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2021 The Trieste Contributors
```

Comment:
lets move the fixtures one level up, to a conftest in a parent folder, now I'm thinking fixtures will be too scattered around, other developers will have hard time finding them

setting the dtype should stay here on the other hand, valid for gpflux only

Reply:
I have moved them. For the `floatx` stuff I have discussed with Andrei and we have decided the best course of action is to have a fixture for setting the `floatx` and then resetting it to the original value, so that this can be done on a test-by-test basis. This uses the pytest yield fixture [https://docs.pytest.org/en/latest/how-to/fixtures.html#teardown-cleanup-aka-fixture-finalization](url)

Reply:
ah, thats a neat solution!